restinparse-core/src/main/java/com/github/jmfayard/ParseQuery.java

```diff
@@ -18,7 +15,7 @@
     public final String className;
     public final Map<String, String> params;
 
-    ParseQuery(String className, Map<String, String> params, Map<String, Object> where, List<String> includes, int skip, int limit, String order) {
+    ParseQuery(String className, Map<String, String> params, Map<String, Object> where, List<String> includes, int skip, int limit, String order, int count) {
```

Comment:
Map<String, Object> where
->
Map<String, ?> where
or something implementing a not yet existing interface?
Map<String, ? extends RIPWhere> where